input_files/strtx_III_dsp/dsp_hb_settings.txt

```diff
@@ -114,9 +118,10 @@ tilehi_tielo_cells_between_power_gnd=True
 #core_site_name=core
 
 #specify the utilization of the core site. you can specify multiple ones on different lines.

```

Comment:
Explain what 0.8 means: 80% of area can be occupied by standard cells 
(I think!)

Reply:
Yes that is correct I'll link this more in depth explanation in comments:
https://community.cadence.com/cadence_technology_forums/f/digital-implementation/48060/what-is-the-meaning-of-cell-utlilization